src/components/common/PaginationControls/style.module.scss

```diff
@@ -1,11 +1,14 @@
+@use 'src/styles/vars.scss' as vars;
+
 .paginationBtns {
   align-items: center;
   display: flex;
-  gap: 20px;
+  gap: 1.5rem;
   justify-content: center;
 
   .paginationBtn {
```

Comment:
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/5bf2cbc7-4c64-47eb-ad75-6109820a0c5a)

maybe you should make these a bit larger so they're the same size as the page numbers

Reply:
actually nvm i like that it;s smaller